main/acle.md

```diff
@@ -2675,10 +2677,12 @@ The following attributes trigger the multi version code generation:
 `__attribute__((target_version("name")))` and
 `__attribute__((target_clones("name",...)))`.
 
+* Functions are allowed to have the same name and signature when
+  annotated with these attributes.
 * These attributes can be mixed with each other.
+* `name` is the dependent features from the tables below.
```

Comment:
Don't use below or above, because they can be ambiguous. Best to cross-reference to the section containing the tables (Mapping?)